pkg/cmd/cli/cmd/rollback.go

```diff
@@ -38,151 +43,267 @@ a human-readable representation of the updated deployment configuration instead
 executing the rollback. This is useful if you're not quite sure what the outcome
 will be.`
 
-	rollbackExample = `  // Perform a rollback
-  $ %[1]s rollback deployment-1
+	rollbackExample = `  // Perform a rollback to the last successfully completed deployment for a deploymentconfig
+  $ %[1]s rollback frontend
 
-  // See what the rollback will look like, but don't perform the rollback
-  $ %[1]s rollback deployment-1 --dry-run
+  // See what a rollback to version 3 will look like, but don't perform the rollback
+  $ %[1]s rollback frontend --to-version=3 --dry-run
+
+  // Perform a rollback to a specific deployment
+  $ %[1]s rollback frontend-2
 
   // Perform the rollback manually by piping the JSON of the new config back to %[1]s
-  $ %[1]s rollback deployment-1 --output=json | %[1]s update deploymentConfigs deployment -f -`
+  $ %[1]s rollback frontend --output=json | %[1]s update deploymentConfigs deployment -f -`
 )
 
 // NewCmdRollback creates a CLI rollback command.
 func NewCmdRollback(fullName string, f *clientcmd.Factory, out io.Writer) *cobra.Command {
-	rollback := &deployapi.DeploymentConfigRollback{
-		Spec: deployapi.DeploymentConfigRollbackSpec{
-			IncludeTemplate: true,
-		},
-	}
-
+	opts := &RollbackOptions{}
 	cmd := &cobra.Command{
-		Use:     "rollback DEPLOYMENT",
+		Use:     "rollback <dc-name|rc-name>",
```

Comment:
*rollback (DEPLOYMENTCONFIG | DEPLOYMENT)* so to be consistent with [other](https://github.com/openshift/origin/blob/08f6608c241e2aa14afd78d82d82de8b290e3cb6/pkg/cmd/cli/cmd/process.go#L49) [commands'](https://github.com/openshift/origin/blob/08f6608c241e2aa14afd78d82d82de8b290e3cb6/pkg/cmd/cli/cmd/deploy.go#L68) [use](https://github.com/openshift/origin/blob/08f6608c241e2aa14afd78d82d82de8b290e3cb6/pkg/cmd/cli/cmd/newbuild.go#L46).